app/views/recipes/show.html.erb

```diff
@@ -0,0 +1,91 @@
+<section id='recipe-details'>
+  <div class='my-5'>
+    <h1 class='text-center underline'><%= @recipe.name %></h1>
+    <hr>
+    <div class='row'>
+      <div class='col-sm-6'>
+        <table class='table table-bordered table-sm'>
+          <tr>
+            <th scope="row">Preparation time</th>
+            <td scope="col"><%= @recipe.preparation_time.to_s + ' min' %></td>
+          </tr>
+          <tr>
+            <th scope="row">Cooking time</th>
+            <td scope="col"><%= @recipe.cooking_time.to_s + ' min' %></td>
+          </tr>
+        </table>
+      </div>
+      <div class='col-sm-6'>
+        <div class='d-flex justify-content-end'>
+          <div class="form-group mt-3">
+            <div class="form-check form-check-inline">
+              <input type="checkbox" class="form-check-input" id="public" 
+              <% if @recipe.public %>
+              checked="checked" 
+              <% end %>
+              />
+              <label class="form-check-label" for="public">Public</label>
+            </div>
+          </div>
+        </div>
+      </div>
+    </div>
+
```

Comment:
- [ ] it would be better if you add a button that will add ingredients to the recipe as required 

On the recipe show page, you should be able to add ingredients.
<img width="1360" alt="Screenshot 2023-02-10 at 17 53 18" src="https://user-images.githubusercontent.com/61790587/218149743-af17b635-4821-4026-b385-4d2e816b0ea1.png">